Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerForMainServer.java

```diff
@@ -0,0 +1,33 @@
+package nl.soccar.gameserver.controller.rmi;
+
+import nl.soccar.gameserver.model.GameServer;
+import nl.soccar.library.enumeration.BallType;
+import nl.soccar.library.enumeration.Duration;
+import nl.soccar.library.enumeration.MapType;
+import nl.soccar.rmi.interfaces.IGameServerForMainServer;
+
+import java.io.IOException;
+import java.rmi.RemoteException;
+import java.rmi.server.UnicastRemoteObject;
+
+/**
+ * @author PTS34A
+ */
+public final class GameServerForMainServer extends UnicastRemoteObject implements IGameServerForMainServer {
+
+    private final GameServerRmiController controller;
+
+    public GameServerForMainServer(GameServerRmiController controller) throws RemoteException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)